tests/conftest.py

```diff
@@ -247,10 +247,9 @@ def is_relevant_file(f):
         return fnmatch.fnmatch(f, "*.id")
 
     for f in filter(lambda f: fnmatch.fnmatch(f, "*.id"), id_files):
-        path = shutil.copyfile(
+        shutil.copyfile(
             f"{PREGENERATED_IDENTITIES_DIR}/{f}", f"{FIXTURE_FILES_DIR}{FIXTURE_FILES_PREFIX}-node_{node_nr}.id"
         )
-        logging.info(f"Copied file {PREGENERATED_IDENTITIES_DIR}/{f} to {path}")
```

Comment:
Why not keeping the log entry?

Reply:
it was added by me when debugging the fixture migration. It wasn't there before. Now it's in the same state as before debugging, but working.